proprietary/tokens/src/components/amsterdam/icon-button.tokens.json

```diff
@@ -0,0 +1,28 @@
+{
+  "amsterdam": {
+    "icon-button": {
+      "background-color": { "value": "transparent" },
+      "color": { "value": "{amsterdam.color.primary-blue}" },
+      "outline-offset": { "value": "{amsterdam.focus.outline-offset}" },
+      "hover": {
+        "background-color": { "value": "rgba(0, 70, 153, 0.13)" },
+        "color": { "value": "{amsterdam.color.dark-blue}" }
+      },
+      "on-background-light": {
```

Comment:
Light is the default, I think we will have a hell of a time when we add a general dark theme because you will have 4 variations in some places and 2 in most.

Reply:
No, see Figma. The default is the white background, ‘light’ means yellow, magenta etc. On white, the icon button background is light blue. On light, it is 12.5% black.